vowpalwabbit/example.h

```diff
@@ -52,6 +52,8 @@ typedef union
 { float scalar;
   uint32_t multiclass;
   MULTILABEL::labels multilabels;
+  float* probs; // for --probabilities --oaa
```

Comment:
This is just a pointer. In oaa.cc, there is both `data.probs = calloc_or_throw<float>(data.k)` and `data.probs = calloc_or_throw<float>(data.k)`. So I hope there are no memory leaks.

Why not using `v_array`? I don't need dynamically growing array. The size is fixed to k (number of classes).

Why not allocating and freeing `probs` within each example? First, I don't see a use case for this (where I would need to access `probs` of all the `ring_size` examples **simultaneously**). Second, I think allocating new memory for each example is slower than reusing one array for all the examples.